### PR TITLE
feat: Base template with no UI library

### LINKED ISSUE #1: feat: Create base react project with no UI library
- [x] Create react project with `pnpm`
- [x] Integration pipeline
- [x] Tagging pipeline
- [x] Release pipeline
- [x] Configure tests with `playwright`
- [x] Configure routes with `react-router-dom`
- [x] Configure `tanstack-query`
- [x] Environment variables support
- [x] Axios singleton instance

### PR DESCRIPTION
**Pull request by PedroChaparro**
## Includes 📋

Base react project with: 

- Integration, tagging and release pipelines. 
- e2e test support 
- Singleton axios instance
- Environment variables support
- Useful packages (`react-router-dom` and `tanstack-query`) configuration.
- `README` file with some instructions
- Pull requests template (The one I'm using in this PR)

## Related Issues 🔎

Closes #1

### CODE REVIEW

**Comment by PedroChaparro** [comment]
I'll create a simple sample project after this gets merged in order to check that everything works as expected

File: version.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+VERSION=$(
+  cat package.json | 
+  grep version | 
+  head -1 |                 
+  awk -F: '{ print $2 }' |  # Split by ":" and get the second part
+  sed 's/[",]//g' |         # Remove quotes and commas
+  tr -d '[[:space:]]'       # Remove spaces
+)
+
+echo "version=$VERSION"
```
Comment (Woynert) [inline version.sh:7]: Nice script! Then I remembered... what about `jq ".version" package.json`?

**Review by Woynert — DISMISSED** [review]
Everything in order, well done :building_construction:

File: version.sh (hunk 1 above)
Comment (PedroChaparro) [reply version.sh:7]: You're right, thanks!

**Pull request merged**